extensions/wikia/FacebookClient/FacebookClient.class.php

```diff
@@ -281,6 +281,19 @@ public function getWikiaUser( $fbId = null ) {
 			return null;
 		}
 
+		// Create a new mapping that includes the app ID.  Leave the default App ID behind
+		// as there is no way to tell what other apps this user had connected, and removing this
+		// default mapping will force the user to reconnect on those apps.
+		if ( $map->isDefaultAppId() ) {
+			$this->migrateMapping( $map );
+		}
+
+		// Update the business token for this user if not already set.  The business token
+		// gives us a unique ID across all apps that we can use to reference a user.
+		if ( !$map->getBizToken() ) {
+			$this->updateBizTokenMapping( $map );
```

Comment:
Nope, ```FacebookMapModel::updateBizToken``` was copied from that classes ```saveToDatabase```.  It doesn't need to throw an error. It should just log that a problem occurred and move on.  I'll remove that ```throw```.